lesson03/assignment/index.js

```diff
@@ -0,0 +1,94 @@
+//JSON data from data.JSON
+[
```

Comment:
You'll want to go ahead and stick this data array in a variable, just add const cards = right before the start of this array and your map function below will work as expected.